src/assets/locales/fr.json

```diff
@@ -156,13 +156,13 @@
   "home-page.favorite-rooms": "Salles Favorites",
```

Comment:
Shouldn't we replace all caps like "Salles Favorites" -> "Salles favorites" 
Same for "Salle" -> "salle" when not at the beginning

Reply:
I was thinking that too. Ok, I will change.